ivr/src/main/java/org/motechproject/ivr/service/OutboundCallService.java

```diff
@@ -14,4 +14,10 @@
      * @param params the parameters that will be used for building the url, the ones that don't match the placeholders will be sent as params of the request
      */
     void initiateCall(String configName, Map<String, String> params);
+
+    /**
+     * Gets the defaultConfig from JSON file and calls the initiateCall method
```

Comment:
It allways take the configuration from the JSON file? Just write that it will use the default configuration